packages/hashi/src/kolibri.js

```diff
@@ -94,6 +94,9 @@ export default class Kolibri extends BaseShim {
        * @param {string[]} options.ids - an array of ids to filter by
        * @param {number} [options.page=1] - which page to return from the result set
        * @param {number} [options.pageSize=50] - the page size for pagination
+       * @param {string} options.random - set to a random string to get the result randomly ordered
```

Comment:
My previous question about why this is a string still stands. To me it should be a "randomizeResults: true/false" boolean.

Reply:
This is because of the cache, the API has a `cacheforever` decorator, so each query will return the same result. To bypass the cache system (if we want real random order with each query) we can use a random string. Using the same string will trigger the cache and will return the same "random" order.

Reply:
Yeah I understand that. But I think the random string should be generated in the backend. For separations of concerns. What do you think?

Reply:
Good idea! I didn't thought about that, maybe we can hide the implementations details in the API and just keep it simple with a boolean.

Reply:
:+1: here is an old but still relevant talk about API design: https://www.youtube.com/watch?v=aAb7hSCtvGw